subprojects/test-runner/service/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ dependencies {
     testImplementation(project(":common:truth-extensions"))
     testImplementation(project(":common:resources"))
     testImplementation(project(":gradle:test-project"))
-    testImplementation(project(":test-runner:shared-test"))
+    testImplementation(project(":test-runner:stub"))
```

Comment:
Do we use it only in tests?
If so, can't we share it by `testFixtures`? Didn't get why we use a standalone module for stubs.

Reply:
we use it in production code, i'll try to move to `device-provider`

Reply:
done in next PR